gravitee-apim-rest-api/gravitee-apim-rest-api-portal/gravitee-apim-rest-api-portal-rest/src/main/java/io/gravitee/rest/api/portal/rest/resource/AbstractResource.java

```diff
@@ -116,28 +107,6 @@ protected boolean hasPermission(RolePermission permission, String referenceId, R
         return isAuthenticated() && (permissionService.hasPermission(permission, referenceId, acls));
     }
 
-    Response.ResponseBuilder evaluateIfMatch(final HttpHeaders headers, final String etagValue) {
```

Comment:
really useless ? Even for picture downloading ?

Reply:
Yes, same in master branch...

Reply:
I think it's copy-paste since avec https://github.com/gravitee-io/issues/issues/2288